proto/hummock.proto

```diff
@@ -50,7 +50,7 @@ message OverlappingLevel {
 message Level {
   uint32 level_idx = 1;
   LevelType level_type = 2;
-  repeated SstableInfo table_infos = 3;
+  repeated SstableInfo sstable_infos = 3;
```

Comment:
These fields are persisted in meta store. Is it safe to rename to break the backward compatibility?

I think changing the in memory struct in `hummock_sdk` should be enough

Reply:
> Is it safe to rename to break the backward compatibility?

Renaming is safe because protobuf identifies fields by their number. I've tested it as well.

Reply:
For metadata stored in etcd backend, we use the protobuf encoding, but for those stored in sql backend, which are encoded as json, I think the fields are identified by their names.

Reply:
blob rather than json is used in sql backend, so backward compatibility is retained.
But since we rarely use pb type directly any longer, this PR will keep pb names unchanged.